packages/core/test/plugins/use-error-handler.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { useErrorHandler } from '../../src/plugins/use-error-handler';
+import { createTestkit } from '@envelop/testing';
+import { buildSchema } from 'graphql';
+import { makeExecutableSchema } from '@graphql-tools/schema';
+
+describe('useErrorHandler', () => {
+  const testError = new Error('Foobar');
+
+  const schema = makeExecutableSchema({
+    typeDefs: /* GraphQL */ `
+      type Query {
+        foo: String
+      }
+    `,
+    resolvers: {
+      Query: {
+        foo: () => {
+          throw testError;
+        },
+      },
+    },
+  });
+
+  it('should invoke error handler when error happens during execution', async () => {
+    const mockHandler = jest.fn();
+    const testInstance = createTestkit([useErrorHandler(mockHandler)], schema);
+    await testInstance.execute(`query { foo }`, {}, { foo: 'bar' });
+
+    expect(mockHandler).toHaveBeenCalledWith([testError], expect.objectContaining({ foo: 'bar' }));
+  });
```

Comment:
Could you also add a test case for when an error is thrown within `useExtendContext`?

Reply:
Nevermind, I was confusing this with useMaskedErrors. All good :)